spring-batch-docs/asciidoc/spring-batch-intro.adoc

```diff
@@ -292,176 +304,176 @@ Batch applications can additionally be categorized by their input source:
+The architecture should be flexible enough to allow dynamic configuration of the number of partitions. Both automatic and user controlled configuration should be considered. Automatic configuration may be based on parameters such as the input file size and the number of input records.
 
 *4.1 Partitioning Approaches*
-		The following lists some of the possible partitioning approaches. Selecting a partitioning approach has to be done on a case-by-case basis.
+		Selecting a partitioning approach has to be done on a case-by-case basis. The following list describes some of the possible partitioning approaches:
 
 _1. Fixed and Even Break-Up of Record Set_
 
-This involves breaking the input record set into an even number of portions (e.g. 10, where each portion will have exactly 1/10th of the entire record set). Each portion is then processed by one instance of the batch/extract application.
+This involves breaking the input record set into an even number of portions (for example, 10, where each portion has exactly 1/10th of the entire record set). Each portion is then processed by one instance of the batch/extract application.
 
-In order to use this approach, preprocessing will be required to split the recordset up. The result of this split will be a lower and upper bound placement number which can be used as input to the batch/extract application in order to restrict its processing to its portion alone.
+In order to use this approach, preprocessing is required to split the recordset up. The result of this split will be a lower and upper bound placement number which can be used as input to the batch/extract application in order to restrict its processing to only its portion.
 
-Preprocessing could be a large overhead as it has to calculate and determine the bounds of each portion of the record set.
+Preprocessing could be a large overhead, as it has to calculate and determine the bounds of each portion of the record set.
 
-_2. Breakup by a Key Column_
+_2. Break up by a Key Column_
 
-This involves breaking up the input record set by a key column such as a location code, and assigning data from each key to a batch instance. In order to achieve this, column values can either be
+This involves breaking up the input record set by a key column, such as a location code, and assigning data from each key to a batch instance. In order to achieve this, column values can be either:
 
-_3. Assigned to a batch instance via a partitioning table (see below for details)._
+_* Assigned to a batch instance by a partitioning table (described later in this section)._
 
-_4. Assigned to a batch instance by a portion of the value (e.g. values 0000-0999, 1000 - 1999, etc.)_
+_* Assigned to a batch instance by a portion of the value (such as 0000-0999, 1000 - 1999, and so on)._
```

Comment:
Is this supposed to be just an * in the rendered output?  It looks odd that it's not a bullet or something...

Reply:
I turned 3 and 4 into bullets under 2. That's why the both start with a lone asterisk.